cfw.md

```diff
@@ -1,9 +1,7 @@
 ---
 layout: page
-title: "BSidesSF 2019 Call For Workshops"
+title: "BSidesSF 2020 Call For Workshops"
 permalink: /cfw.html
 --- 
 
-The BsidesSF 2019 CFW is now closed. BSidesSF workshops will take place in San Francisco on March 2, 2019.
-
-As always -- Questions? E-mail workshops [at] bsidessf.org
+Please see details for submitting a 2020 Workshop at our [Call for Participation](https://bsidessf.org/cfp.html)
```

Comment:
`[Call For Participation](/cfp)`, but is this needed with the redirect?

Reply:
I added it as a fallback just in case something went sideways with the redirect plugin. I don't think that it's strictly necessary, though. I'll add the capitol `F` just in case.

Reply:
The main issue here is changing `https://bsidessf.org/cfp.html` to `/cfp` or `/cfp.html` (all relative links).

Reply:
Capitol F added